docs/writing-modules.adoc

```diff
@@ -77,13 +77,13 @@ foo.bar = {
 +
 would place `c` before `b` and after `a` in the graph.
 
-`hm.types.gvariant`::
-This type is useful for options representing {gvariant-description}[GVariant] values. The type accepts all primitive GVariant types as well as arrays and tuples. Dictionaries are not currently supported.
+[[sec-option-types-gvariant]]`hm.types.gvariant`::
+This type is useful for options representing {gvariant-description}[GVariant] values. The type accepts all primitive GVariant types as well as arrays, tuples, maybe types and dictionaries.
```

Comment:
```suggestion
This type is useful for options representing {gvariant-description}[GVariant] values. The type accepts all primitive GVariant types as well as arrays, tuples, maybe-types and dictionaries.
```

Or anything else to avoid reading "maybe types" literally

Reply:
I put it into quotes.